deploy/cr.yaml

```diff
@@ -51,7 +51,14 @@ spec:
           bucket: S3-BACKUP-BUCKET-NAME-HERE
           credentialsSecret: cluster1-s3-credentials
           region: us-west-2
+
+  router:
+    image: perconalab/percona-server-mysql-operator:router
```

Comment:
we need to have affinity and resources sections for router

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/91/commits/7a939c9b4b475549bd2f461f2334c07ebf35a249